src/utils.tsx

```diff
@@ -1,18 +1,21 @@
 import React, { Children, ReactNode } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { View } from 'react-native';
 
-export const childrenWithOverriddenStyle = (children?: ReactNode) => {
+export const childrenWithOverriddenStyle = (
+  children?: ReactNode,
+  pageMargin = 0
+) => {
   return Children.map(children, (child) => {
-    const element = child as React.ReactElement;
     return (
-      // Add a wrapper to ensure layout is calculated correctly
-      <View style={StyleSheet.absoluteFill} collapsable={false}>
-        {/* @ts-ignore */}
-        {React.cloneElement(element, {
-          ...element.props,
-          // Override styles so that each page will fill the parent.
-          style: [element.props.style, StyleSheet.absoluteFill],
-        })}
+      <View
```

Comment:
I think we need to pass there `child.key` so the outer view has proper key and react can manage it's views efficiently. I remember that's what @satya164 mentioned on one of our internal meetings.